specs/networking/libp2p-standardization.md

```diff
@@ -117,11 +141,13 @@ There are two main topics used to propagate attestations and beacon blocks to
 all nodes on the network.
 
 - The `beacon_block` topic - This topic is used solely for propagating new
-	beacon blocks to all nodes on the networks.
+	beacon blocks to all nodes on the networks. Blocks are sent in their
+	entirety. Clients who receive blocks on a topic SHOULD validate the block
+	before forwarding it across the network.
 - The `beacon_attestation` topic - This topic is used to propagate
-	aggregated attestations to subscribing nodes (typically block proposers) to
-	be included into future blocks. Attestations are aggregated in their
-	respective subnets before publishing on this topic.
+	aggregated attestations (in their entirety) to subscribing nodes (typically
+	block proposers) to be included into future blocks. Attestations are
+	aggregated in their respective subnets before publishing on this topic.
 
 Shards are grouped into their own subnets (defined by a shard topic). The
 number of shard subnets is defined via `SHARD_SUBNET_COUNT` and the shard
```

Comment:
`shard{shard_number % SHARD_SUBNET_COUNT}_attestation` topic is going to have a potential name collision in phase 1 because `ShardAttestation`s will also need to be propagated in addition to the beacon `Attestation` defined in phase 0.

Maybe we should go with `shard{shard_number % SHARD_SUBNET_COUNT}_beacon_attestation` to clearly distinguish between the two

Reply:
Yep sure. I guess that `ShardAttestation`'s will have their own set of gossipsub subnets right?

Reply:
yes